desktop/plugins/network/index.tsx

```diff
@@ -385,6 +394,67 @@ export function plugin(client: PluginClient<Events, Methods>) {
 
         informClientMockChange(routes.get());
       },
+      importRoutes() {
+        const options: OpenDialogOptions = {
+          properties: ['openFile'],
+          filters: [{extensions: ['json'], name: 'Flipper Route Files'}],
+          title: 'Import Routes',
+        };
+        remote.dialog.showOpenDialog(options).then((result) => {
+          const filePaths = result.filePaths;
+          if (filePaths.length > 0) {
+            fs.readFile(filePaths[0], 'utf8', (err, data) => {
+              if (err) {
+                console.error(err);
```

Comment:
nit: you could use `import {message} from "antd"; message.error(...)` here as it gives more visibility to the user, see https://ant.design/components/message/,  but this is fine as well.

Reply:
Completed.  Here's the screenshot showing where the error message displays:

![image](https://user-images.githubusercontent.com/337874/105762832-9051cb00-5f1a-11eb-85b0-4566515a910d.png)